iterative/resource_runner.go

```diff
@@ -380,7 +380,7 @@ func provisionerCode(d *schema.ResourceData) (string, error) {
 				Attributes: AttributesType{
 					ID:                 d.Id(),
 					Cloud:              d.Get("cloud").(string),
-					Region:             utils.StripAvailabilityZone(d.Get("region").(string)),
+					Region:             d.Get("region").(string),
```

Comment:
this will fail if we set `us-west-1a` we wont be able to create the client. We have to use the `utils.GetRegion(d)`

Reply:
That is how I had it originally, I assume you got an error with it? I'm slow and haven't tested it myself yet 🙈 , I was thinking it was safe / I'll swap it back.